src/main/scala/Core/core.scala

```diff
@@ -31,6 +31,12 @@ object FlexpretConfiguration {
       parsed.get.group(5)
     )
   }
+
+  def defaultConfig: FlexpretConfiguration = {
```

Comment:
What is this for?

Reply:
Nice to have a default config if no command line arguments are passed to `sbt run`

Reply:
Please add to scaladoc?